### ISSUE TITLE
false positive empty_count violation

### ISSUE THREAD

**Issue by garnett** [realm/SwiftLint#884]
```
let count = try! appDelegate.persistentContainer.viewContext.count(for: fetchRequest)

if count == 0 { // swiftlint error
}
```

Workaround is to rename variable to something different, i.e. `entitiesCount`

**Comment by jpsim** [comment]
Please see https://github.com/realm/SwiftLint/issues/827#issuecomment-263009822.
